docs/pretty.md

```diff
@@ -23,8 +23,13 @@ The API requires modules provide a factory function which returns a prettifier
 function. This prettifier function must accept either a string of NDJSON or
 a Pino log object. A psuedo-example of such a prettifier is:
 
+Pino instance is passed as `this` into prettifier factory function. But it 
```

Comment:
```suggestion
The uninitialized Pino instance is passed as `this` into prettifier factory function, so it can be accessed via closure by the returned prettifier function.
```